game/game.go

```diff
@@ -800,19 +977,73 @@ func AllocateNations(preferers Preferers, nations godip.Nations) ([]godip.Nation
 	return result, nil
 }
 
+func (g *Game) ValidNation(nation godip.Nation) bool {
+	for _, nation := range variants.Variants[g.Variant].Nations {
+		if nation == nation {
```

Comment:
You need to change either the variable name or the parameter name.

Reply:
Thanks!